### ISSUE TITLE
ASA: Zonal Head Fire Intensity statistics - Percentage of the combustible land area

### ISSUE THREAD

**Issue by nealmcloughlin** [bcgov/wps#3205]
**Describe the task**
Clicking on a fire zone boundary on the ASA map opens a side panel with statistics. The first chart summarizes the percentage of the combustible land area according to two expected fire intensity classes: 4,000 to 9,999 kW/m and >= 10,000 kW/m. This chart could use some UX refinement.

**Acceptance Criteria**
- [x] X-axis labels should align with the two intensity classes.
- [x] Y-axis is dynamic and re-scales when you click on different fire zones. This can be visually misleading if a person doesn't pay close attention to the Y-axis is focused more on the height of the bars, i.e.) 14% can have the same height as 40% when the Y-axis re-scales.
- [x] The Y-axis title overlaps with the labels.
- [x] The color of the bars could align with how the two intensity classes appear on the map.
- [x] The chart title should refer to Fire Intensity classes, not Advisory/Warning.

- [x] Ensure that the percentage of the combustible land area is stated clearly

**Additional context**
- We can reference fire intensity classes if to avoid the technical kW/m ranges. There are even pictures of what the different fire intensity classes look like.

**Comment by rajpersram** [comment]
@nealmcloughlin , are these ranking relevant to the Warning or Advisory status? https://www2.gov.bc.ca/gov/content/safety/wildfire-status/wildfire-response/about-wildfire/wildfire-rank

**Comment by rajpersram** [comment]
@nealmcloughlin Do we want to include classes of HFI (1-5) as noted on this Canadian Federal site?
https://cfs.nrcan.gc.ca/pubwarehouse/pdfs/24358.pdf

**Comment by rajpersram** [comment]
Food for thought and team discussion: It might be worth considering acceptance criteria more as a set of desired outcomes, which leaves room for how that might be manifested, rather than trying to be too prescriptive on how it should be manifested. That will leave room for reconsidering the current user interface. 

In this ticket's context, if we're aiming at users being able to understand, at a glance, what the percentage of the combustible land as intuitively and clearly as possible, that might entail abandoning the bar chart altogether and simply showing a text percentage. In that case, the acceptance criteria above wouldn't necessarily apply. @NicolasLivanos, what do you think as a Scrum Master about this approach?

**Comment by nealmcloughlin** [comment]
@rajpersram The wildfire ranks are relevant to Warning and Advisory status. They are not exactly the same intensity classes but essentially communicate the same information. I'll have to go into my course notes to explain the technical difference between wildfire rank and intensity class. We could report the percentage of combustible land in each HFI class. However, anything below HFI class 5 does not meet the criteria for an Advisory or Warning.

**Comment by nealmcloughlin** [comment]
@rajpersram Head Fire Intensity (HFI) classes refer to energy output ranges in kW per meter of fire front. These numbers are hard to identify without a visual aid. BCWS created wildfire ranks to give fire fighters visual fire behaviour characteristics that can be used to determine appropriate fire fighting tactics. The six ranks essentially align with the six HFI classes. We can consider them synonymous with respect to fire behaviour. In terms of messaging, Ranks allow us to speak to fire fighting tactics.

**Comment by nealmcloughlin** [comment]
@rajpersram I agree with your comment regarding acceptance criteria. My bad for being too prescriptive. That was not my intent. I just wanted to capture some of the ideas we discussed. I should have put those bullets under additional context and left the acceptance criteria for refinement with the team.

**Comment by rajpersram** [comment]
'Percentage of Combustible Land' title changed to 'Portion of Zone under Advisory'.

**Comment by rajpersram** [comment]
Related dev tickets:
[Summarize the Percentage of Combustible Land #2448](https://github.com/bcgov/wps/issues/2448)
[Zonal summary of Head Fire Intensity #2544](https://github.com/bcgov/wps/issues/2544)
[Additional Grids: Elevation - UX #2190](https://github.com/bcgov/wps/issues/2190)
